DEVELOPMENT_GUIDE.rst

```diff
@@ -70,6 +70,13 @@ drops below 95%.
 
 ``pytest -k "TestMyClass"`` command will run all unit tests within the `TestMyClass` class.
 
+End to end tests
+~~~~~~~~~~~~~~~~
+
+``make e2e tests`` command will run all end to end tests. This command is configured to fail when code coverage for package drops below 95%.
+
+``pytest end-to-end-tests/tst/test.py`` command will run the integration test `test.py`
+
 Pull Requests
 -------------
 Before sending pull requests make sure to run ``make pr`` command. This will run unit tests, linters, and static
```

Comment:
Do we still only want `make pr` to work? Should running integ tests be apart of `make pr`?

Reply:
I have added [here](https://github.com/awslabs/serverless-application-model/pull/1557/files#diff-195a635ad245ded9076330e31134bd80R12) but forgot to add in this page. I will update it
I didn't add a way to running integ tests as part of build but we will eventually want integ tests to be a part of make pr.